presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,81 @@
+
+## Web UI
+
+* Add support for OAuth2 authorization in Web UI. ({issue}`5355`)
+* Fix invalid operator stats being reported in stage performance view. ({issue}`6114`)
+
+## JDBC Driver Changes
+
+* Allow reading `timestamp with time zone` value as `ZonedDateTime` using `ResultSet.getObject(int column, Class<?> type)` method. ({issue}`307`)
+* Change representation of a `row` value. `ResultSet.getObject` now returns an instance of `io.prestosql.jdbc.Row` class, which better represents
+  the returned value. Previously a `row` value was represented as a `Map` instance, with unnamed fields being named like `field0`, `field1`, etc. 
+  You can access the previous behavior by invoking `getObject(column, Map.class)` on the `ResultSet` object. ({issue}`4588`)
+* Represent `varbinary` value using hex string representation in `ResultSet.getString`. Previously the return value was useless, similar to `"B@2de82bf8"`. ({issue}`6247`)
+* Consider session properties when caching metadata. ({issue}`6167`)
+* Fix failure when reading a `timestamp` or `timestamp with time zone` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
+* Fix failure when reading a `time` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6204`)
+* Fix element representation in arrays returned from `ResultSet.getArray`, making it consistent with `ResultSet.getObject`.
+  Previously the elements were represented using internal client representation (e.g. `String`). ({issue}`6048`)
+* Fix `ResultSetMetaData.getColumnType` for `timestamp with time zone`. Previously the type was miscategorized as `java.sql.Types.TIMESTAMP`. ({issue}`6251`)
+* Fix failure when an instance of `SphericalGeography` Geospatial type is returned in the `ResultSet`. ({issue}`6240`)
+* Fix cache entries invalidation, so entries are removed for all users not only the one that was running the query. ({issue}`6167`)
+
+## CLI Changes
+
+* Fix rendering of `row` values with unnamed fields. Previously they were printed using fake field names like `field0`, `field1`, etc. ({issue}`4587`)
+* Fix query progress reporting. ({issue}`6119`)
+* Fix failure when an instance of `SphericalGeography` Geospatial type is returned to the client. ({issue}`6238`)
+
+## Hive Connector Changes
+
+* Reduce scheduling latency for queries where scanned files that are referenced by symlinks
+  are located in the same directory. ({issue}`6158`)
+* Add deserializer class name to hive connector split information, which is exposed through the event listener. ({issue}`6006`)
+* Improve parallelism for queries where scanned files are referenced by symlinks. ({issue}`6213`)
+* Add support for S3 streaming uploads. Data is uploaded to S3 as it is written, rather
+  than staging to a local temporary file. This feature is disabled by default and can be enabled
+  using the `hive.s3.streaming.enabled` configuration property. ({issue}`3712`, {issue}`6201`)
```

Comment:
also .. is this property documented in https://prestosql.io/docs/current/connector/hive-s3.html ?